assets/js/modules/adsense/components/settings/AdBlockingRecoveryApp/steps/CreateMessageStep.js

```diff
@@ -129,7 +167,11 @@ export default function CreateMessageStep() {
 									'google-site-kit'
 								) }
 							</SpinnerButton>
-							<Link href={ privacyMessagingURL } target="_blank">
+							<Link
+								onClick={ handleSecondaryCTAClick }
+								href={ privacyMessagingURL }
+								target="_blank"
```

Comment:
If the intention here is to be an external link with no indicator, we have props for that (`external` and `hideExternalIndicator`).

Reply:
Are you asking about the `target="_blank"`? I only added the callback here, it was like this previously. I will replace the target blank with your suggested props as it seems like the right think to do. Cheers.